style.css

```diff
@@ -0,0 +1,467 @@
+header {
+  width: 100%;
+  height: 60px;
+  padding: 0 16px;
+  background-color: rgb(245, 235, 235);
+  position: fixed;
+  top: 0;
+}
+
+nav {
+  display: flex;
+  height: 100%;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.start-pt {
+  display: flex;
+  height: 100%;
+  align-items: center;
+}
+
+.bars-info {
+  margin-right: 16px;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+}
+
+.nav-brand {
+  width: 80px;
+}
+
+.center {
+  display: flex;
+  width: 40%;
+  height: 100%;
+  align-items: center;
+}
+
+.search-bar {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  border: 1px solid;
+  border-color: lightgray;
+}
+
+.search-bar input[type=text] {
+  flex: 1;
+  padding: 4px 6px;
+  font-size: 16px;
+  font-family: "Roboto", "Noto", sans-serif;
+  font-weight: 400;
+  line-height: 24px;
+  border: none;
+  border-right: 1px solid;
+  border-color: lightgray;
+}
+
+.search-bar button {
+  padding: 4px 8px;
+  background: #f8f8f8;
+  border: none;
+  cursor: pointer;
+}
+
+.nav-right {
+  display: flex;
+  align-items: center;
+  margin-right: 32px;
+}
+
+.nav-right button {
+  margin-right: 8px;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+}
+
+.nav-ico {
+  width: 32px;
+  height: 32px;
+}
+
+.nav-right a img {
+  width: 32px;
+  height: 32px;
+  border-radius: 50%;
+}
+
+main {
+  margin-top: 66px;
+  display: flex;
+  justify-content: center;
+  background-color: #f9f9f9;
+}
+
+.video-content {
+  width: 30%;
+}
+
+.video-player {
+  position: relative;
+  padding-bottom: 45%;
+  margin-top: 30px;
+  padding-top: 25px;
+  margin-right: 30px;
+  margin-left: 80px;
+  width: 90%;
+}
+
+.video-player iframe {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 95%;
+  height: 100%;
+}
+
+.vid-info {
+  padding: 20px 0 10px 0;
+  border-bottom: 1px solid;
+  border-color: lightgray;
+  margin-left: 80px;
+  font-size: 12px;
+}
+
+.btn-thumbs {
+  position: relative;
+  font-size: 10px;
+  left: 465px;
+  bottom: 30px;
+  margin-right: 180px;
+  cursor: pointer;
+}
+
+.line {
+  margin-right: 60px;
+}
+
+.text-info {
+  font-size: 14px;
+  font-weight: 400;
+  letter-spacing: normal;
+  line-height: 21px;
+  color: #606077;
+}
+
+.chanel-desc {
+  display: flex;
+  align-items: center;
+  margin-left: 80px;
+}
+
+.pic {
+  width: 48px;
+  margin-right: 16px;
+  border-radius: 50%;
+  cursor: pointer;
+}
+
+.channel-subs {
+  position: absolute;
+  font-size: 12px;
+  left: 145px;
+  margin-top: 70px;
+}
+
+.chan-desc {
+  font-size: 14px;
+  font-weight: 500;
+  letter-spacing: normal;
+  line-height: 16px;
+  margin-left: 64px;
+}
+
+.channel-description {
+  margin-left: 75px;
+  color: rgb(96, 96, 96);
+  font-size: 13px;
+  font-weight: 400;
+  line-height: 18px;
+}
```

Comment:
Please add the right font size and right margins to "subscribe" to your design and try to stick as much as possible to the given design.
kindly Increase font size of description page
![font](https://user-images.githubusercontent.com/97598663/159271289-404960d0-fa57-498d-b246-b50e4315386b.jpg)